frontend/src/metabase/admin/tasks/containers/Help.jsx

```diff
@@ -58,13 +58,6 @@ function githubIssueLink(bugReportDetails) {
   );
 }
 
-function discourseLink(bugReportDetails) {
```

Comment:
Update the help page according to the design https://www.notion.so/metabase/Make-it-easier-to-get-help-7302f0ddeecb49a8a0ed990a53dcc977#974bea7244fc4fa49fd5436e4f821b98